lib/rust/parser/src/macros/expand.rs

```diff
@@ -5,6 +5,9 @@ use crate::prelude::*;
 
 use crate::syntax;
 
+use std::collections::HashMap;
+use std::collections::HashSet;
```

Comment:
Arent these already imported by prelude?

Reply:
To reduce the size of Prelude (and eventually remove it) that was removed.

Reply:
Hmm, and do we aim for prelude _removal_? I assumed it should be just slimmed down.